src/AccessibilityInsights.SharedUx/Resources/Styles.xaml

```diff
@@ -16,7 +16,7 @@
             <Setter.Value>
                 <ControlTemplate>
                     <Border BorderThickness="2" Margin="-2"
-                      BorderBrush="{DynamicResource ResourceKey=IconBrush}"
+                      BorderBrush="{DynamicResource ResourceKey=FocusBorderBrush}"
```

Comment:
This is to decouple the focus highlight from the icon brush color (was needed only for HC White)